utils/data-processing-family-task-submitter-container/pom.xml

```diff
@@ -0,0 +1,162 @@
+                            <filter>data-processing-family-task-submitter</filter>
+                        </configuration>
+                    </execution>
+                </executions>
+                <configuration>
+                <autoPull>always</autoPull>
+                    <images>
+                        <!-- Task Submitter Container Image -->
+                        <image>
+                            <alias>data-processing-family-task-submitter</alias>
+                            <name>${cafFamilyTaskSubmitterContainerName}</name>
+                            <build>
+                                <from>cafapi/opensuse-jre8:1.2</from>
+                                <labels>
+                                    <Build.Number>${project.version}</Build.Number>
+                                    <Build.Date>${maven.build.timestamp}</Build.Date>
+                                    <Git.Branch>${git.branch}</Git.Branch>
+                                    <Git.Commit>${git.revision}</Git.Commit>
+                                </labels>
+                                <optimise>true</optimise>
+                                <assembly>
+                                    <basedir>/</basedir>
+                                    <mode>tar</mode>
+                                    <inline>
+                                        <dependencySets>
+                                            <dependencySet>
+                                                <useProjectArtifact>true</useProjectArtifact>
+                                                <includes>
+                                                    <include>com.github.cafdataprocessing:data-processing-family-task-submitter</include>
+                                                </includes>
+                                                <outputFileNameMapping>family-task-submitter.jar</outputFileNameMapping>
+                                            </dependencySet>
+                                            <dependencySet>
+                                                <useProjectArtifact>false</useProjectArtifact>
+                                                <fileMode>0755</fileMode>
+                                                <includes>
+                                                    <include>com.github.krallin:tini</include>
+                                                </includes>
+                                            </dependencySet>
+                                        </dependencySets>
+                                    </inline>
+                                </assembly>
+                                <entryPoint>
+                                    <exec>
+                                        <args>/tini-${tini.version}.exe</args>
+                                        <args>--</args>
+                                    </exec>
+                                </entryPoint>
```

Comment:
You can drop the `entrypoint` and `tini` will be automatically used.

Reply:
Removed.